### PR TITLE
rMscreens

### LINKED ISSUE #568: Add rmscreens package
I wrote a small utility to easily change the remarkable sleep screen and other screens via CLI.  It is composed of 2 scripts.  The first script sits on the rM that accepts 2 arguments 1. the screen you want to change and 2. the location of the new .png.  The second script sits on the local computer and asks a series of questions and passes the needed arguments to the first script to change the screen on the rM.

https://github.com/pr0fsmith/rMscreens

GNU General Public License.

This is my first time ever doing this and my first time ever writing a bash script so your feedback would be appreciated

### PR DESCRIPTION
**Pull request by pr0fsmith**
rMscreens is a commandline utility that allows you to easily change your rM screen (sleep screen, power off screen etc).  It can also be used to backup and restore your original screens easily.

I wrote this code myself and tested it on the rM2 software version 2.12 https://github.com/pr0fsmith/rMscreens/blob/main/changescrn

### CODE REVIEW

**Comment by Eeems** [comment]
I would really recommend adding help output to the script.

I don't think we can merge this until https://github.com/pr0fsmith/rMscreens/issues/2 is resolved though.

**Review by Eeems — CHANGES_REQUESTED** [review]
Move backup location off of root partition

**Comment by Eeems** [comment]
@pr0fsmith did you have a chance to address the requested changes?

**Comment by pr0fsmith** [comment]
> @pr0fsmith did you have a chance to address the requested changes?

I will do that today.

**Comment by pr0fsmith** [comment]
@Eeems All changes have been made.

**Comment by Eeems** [comment]
@pr0fsmith I don't see any changes to this PR? Did you intend to update the source and sha256sum in the package file to pull in the latest changes?

**Comment by pr0fsmith** [comment]
> @pr0fsmith I don't see any changes to this PR? Did you intend to update the source and sha256sum in the package file to pull in the latest changes?

@Eeems I actually didn't make any changes. It seems as though I made the changes months ago so the existing source should be fine.

**Comment by Eeems** [comment]
@pr0fsmith it looks like this is still pointing at an old commit and doesn't have the changes that were requested.

**Comment by Eeems** [comment]
@pr0fsmith poke

**Comment by pr0fsmith** [comment]
I want to but I honestly dont understand how. I find the whole process confusing.

Sept 26, 2022 10:57:48 a.m. Nathaniel van Diepen ***@***.***>:

File: package/rmscreens/package [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+pkgdesc="Change sleep/suspend/poweroff/reboot screens"
+url=https://github.com/pr0fsmith/rMscreens/
+pkgver=1.0.0-1
+timestamp=2022-03-15T01:28:33Z
+section="utils"
+maintainer="pr0fsmith <evon_smith@yahoo.ca>"
+license=GPL-3.0
+
+source=(https://github.com/pr0fsmith/rMscreens/archive/0775fa67ace15cfce85e07ceda38397dfcee8236.zip)
+sha256sums=(7ecf9c5355a5f06a3424fe24b04e94fc458b93b83ca015e5700308f683d8c102)
```
Comment (Eeems) [inline package/rmscreens/package]:
```suggestion
source=(https://github.com/pr0fsmith/rMscreens/archive/47c01faf51cbb8dd6a83c5a26e646ae246ee26f6.zip)
sha256sums=(49cb4c57556ed807d6de613c63874f1253e54989f0445439a06158abd8514bc3)
```

**Review by Eeems — CHANGES_REQUESTED** [review]

**Comment by Eeems** [comment]
@pr0fsmith I've added a comment showing what needs to be changed. There is a button to directly apply the suggested changes.

Updates to your repository don't automatically show up here as we point at specific commits so that we know that the source doesn't change between builds. Whenever you update your repository you need to point the recipie at the new commit.

**Comment by pr0fsmith** [comment]
I don't understand how to change the recipe. You did it for me last time.

**Comment by Eeems** [comment]
> I don't understand how to change the recipe. You did it for me last time.

If you go to the files changed tab, you'll see the recipie there. It's the `package/rmscreens/package` file.

**Comment by Eeems** [comment]
@pr0fsmith I can't get this to work, there is no help output. When looking at the comment in the script saying how to use it, that doesn't work. It looks like your argument parsing has a loop where it will never work as both -c and -n expect the other setting to already be set.
image

**Comment by Eeems** [comment]
@pr0fsmith any chance you could get this cleaned up and ready? I'd love to get it merged.

**Review by Eeems — APPROVED** [review]

**Comment by Eeems** [comment]
It would be nice to have this automatically do a backup/restore on install, but we'd need a better mechanism to make sure that normal usage can't interfere with what files are backed up or restored.

**Pull request merged**